src/Controller/DemoController.php

```diff
@@ -91,8 +91,6 @@ private function processFormAndReturnRoute(FormInterface $form): RedirectRespons
         $config = $demoFormData->getConfig();
 
         $rectorRun = $this->createRectorRun($config, $demoFormData);
-        $this->rectorRunRepository->save($rectorRun);
-
```

Comment:
This one save() is not needed, right? It's duplicate of the line bellow

Reply:
Yeah it is no longer required after the refactorings!

Reply:
Yeah it is no longer required after the refactorings!

Reply:
Thanks for confirmation